virt/getting_started/virt-web-console-overview.adoc

```diff
@@ -898,18 +897,29 @@ Restart the virtual machine to apply changes.
 |*Cloud-init*
 |Click the edit icon to edit the cloud-init settings.
 
-|*Public SSH key*
-|Click the edit icon to add a public SSH key to a Linux virtual machine.
+|*Sysprep*
+|Click the edit icon to upload an `Autounattend.xml` or `Unattend.xml` answer file to automate Windows virtual machine setup.
+|====
+=====
 
-The key is added as a cloud-init data source at first boot.
+[id="virtualmachine-details-metadata_virt-web-console-overview"]
+===== Metadata tab
 
-|*Dynamic SSH key injection* switch
-|Set *Dynamic SSH key injection* to on to enable dynamic public SSH key injection. Then, you can add or revoke the key at runtime.
+You configure the labels and annotations on the *Metadata* tab.
 
-Dynamic SSH key injection is only supported by {op-system-base-full} 9. If you manually disable this setting, the virtual machine inherits the SSH key settings of the image from which it was created.
+.*Metadata* tab
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+|====
+|Element
+|Description
+
+|*Labels*
+|Click the edit icon to manage your labels.
 
 |*Sysprep*
```

Comment:
Is Sysprep just a typo? 

```suggestion
|*Annotations*
```

Reply:
In this particular section, the two UI elements are *Labels* and *Annotations*, so we thought it was safe to assume that "Sysprep" was a typo left over from the previous table :) It should be changed to *Annotations* here.

Reply:
ohhh good catch!